src/components/inputs.js

```diff
@@ -0,0 +1,20 @@
+// @flow
```

Comment:
inputs.js ? why plural form?

Reply:
I wanted to use some inputs. But now I'm not sure. I will change.